sys-cluster/charliecloud/charliecloud-0.9.10.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
```

Comment:
Any chance you could install 3.7 and test it? We're going to do the switch sooner than later.

Reply:
Done just now. The newly pushed version also has the compat for python 3.7.